lib/css/exports/_stacks-mixins.less

```diff
@@ -29,6 +29,27 @@
             appearance: @all;
 }
 
+//  ===========================================================================
+//  --  FOCUS VISIBLE
+//      Powered by https://github.com/WICG/focus-visible while we wait for better
+//      :focus-visible support
+//  ---------------------------------------------------------------------------
+.focus(@rules) {
```

Comment:
Should this be `focus-visible` or some variant thereof? `focus` already has its own different meaning.